.github/workflows/linux.yml

```
@@ -42,8 +42,17 @@ jobs:
     #
     # if: ${{ github.event_name == 'push' || !github.event.pull_request.draft }}
 
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - os: ubuntu-22.04
+          image: dealii/dependencies:jammy
+        - os: ubuntu-24.04
+          image: masterleinad/dependencies-dealii:noble
```

```suggestion
          image: dealii/dependencies:noble
```
This is now up-to-date